forwards_paper.tex

```diff
@@ -115,14 +119,15 @@ \section*{Introduction}
 in a population having even hundreds of millions of individuals.
 However, if our interest lies in the resulting genetic patterns of variation
 -- and often, the point of such simulations is to compare to real data --
-then such simulations must somehow produce at the end data for each individual on a genomic scale.
+then such simulations must record data for each individual genome.
 As samples of most species' genomes harbor tens or hundreds of millions of variant sites,
 naively carrying full genotypes for even modest numbers of individuals through a simulation
 becomes quickly prohibitive.
 To make matters worse,
 a population of size $N$ must be simulated across many multiples of $N$ generations
-to produce stable genetic patterns.
-Because of this computational burden of neutral mutations, even the fastest simulation frameworks such as SLiM,
+to produce stable genetic patterns \jda {citation?}.
```

Comment:
Er, maybe
```
@book{wakeley2005coalescent,
  address = {Greenwood Village, CO},
  author = {Wakeley, John},
  biburl = {https://www.bibsonomy.org/bibtex/2c6c6b280ef6b6d13e4ad56600381f1d2/peter.ralph},
  keywords = {coalescent_theory reference},
  publisher = {Roberts and Company},
  title = {Coalescent Theory, an Introduction},
  url = {http://www.coalescentheory.com/},
  year = 2005
}
```

Reply:
Should cite SLiM2 not SLiM here (former about 6x faster), and fwdpp (about another 6-8x faster still).

For times to equilibrium, the key citations will be earlier, probably Crow & Kimura or Kimura's mid-1950s treatment on the diffusion?  But the results probably pre-date all of that, going back to Fisher and Wright.

Reply:
was aiming to cite a recent readable statement of that fact; but don' tthink we necessarily need to have a citation.  This is discussed in Wright, Evolution in Mendelian Populations, Genetics 16 1931, though.

Reply:
Oops, my fault there was a citation to both just below that I mangled in this edit